src/pages/landing/filters/status/StatusFilterBody.tsx

```diff
@@ -49,27 +57,57 @@ export const StatusFilterBody = ({
       {...rest}
     >
       {options.map((option, index) => {
+        if (option.linkTo) {
+          return (
+            <Link key={index} to={option.linkTo}>
+              <Button
```

Comment:
you can pass in: as={Link} and to ={..} to the Button itself, no need to wrap the button, it'll make it neater.